demos/classification_benchmark_demo/cpp_gapi/classification_benchmark_demo_gapi.hpp

```diff
@@ -26,7 +31,14 @@ static const char no_show_message[] = "Optional. Disable showing of processed im
 static const char execution_time_message[] = "Optional. Time in seconds to execute program. "
                                              "Default is -1 (infinite time).";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
-
+#ifdef GAPI_IE_EXECUTION_PROVIDERS_AVAILABLE
+static const std::string ep_message_str("Optional. List of inference execution providers: " +
+                                    merge(getSupportedEP(), ",") +
+                                    " - separated by \";\". Each provider followed by its device. Use \":\" as <provider:device> separator");
```

Comment:
I've got a reply only to the first part.

> Please use , instead of ; to stay consistent with the rest lists, like -d

is still applicable. I just realized that using the same separators as OpenVINO uses: `:`, `,` can lead to ambiguity in `-ep DML:CPU;OVEP:HETERO:GPU,CPU`. So instead you may want to use completely different separators. I'm OK with any of two solutions: using only `:`, `,`, or using completely different ones. But please, don't use a mix of this two approaches.